.browserslistrc

```diff
@@ -0,0 +1 @@
+last 2 versions
```

Comment:
That's what we had for the CSS (autoprefixer)... We could maybe change that? Using a file allows to configure ALL tools that rely on browserslist.